addons/web/static/src/views/fields/file_handler.js

```diff
@@ -47,7 +47,9 @@ export class FileUploader extends Component {
                 this.state.isUploading = false;
             }
         }
-        ev.target.value = null;
+        if (ev.target) {
+            ev.target.value = null;
+        }
```

Comment:
This is not the correct fix. `input.value = null` is important so that you can re-upload another file with same name.

The problem comes from lookup on `ev` after some async code. The correct fix is to save `input` reference before the async behaviour. Any data in `ev` is reliable before async behaviour.

Reply:
Not required anymore, reverted :+1: